src/site/stages/build/process-cms-exports/transformers/node-page.js

```diff
@@ -8,13 +8,23 @@ const {
   isPublished,
 } = require('./helpers');
 
+const getFieldAlert = fieldAlert => {
+  if (!isEmpty(flatten(fieldAlert))) {
+    return fieldAlert[0];
+  }
+  return isEmpty(flatten(fieldAlert)) && isEmpty(fieldAlert)
```

Comment:
I think all you need here is `isEmpty(fieldAlert)` in the condition since you escape early above anyways 🙂

Reply:
Ahh nice catch!